hack/internal/pkg/utils/clone.go

```diff
@@ -0,0 +1,112 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	ctrlcommon "github.com/openshift/machine-config-operator/pkg/controller/common"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/openshift/machine-config-operator/test/framework"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+)
+
+const (
+	ClonedObjectLabelKey      string = "machineconfiguration.openshift.io/cloned-by-zacks-openshift-helpers"
```

Comment:
issue: We probably should change this to something like `machineconfiguration.openshift.io/cloned-by-mco-helpers` instead.